tests/xtro-sharpie/common-AppKit.ignore

```diff
@@ -1,857 +1,16 @@
 ## NSUserInterfaceItemIdentifier -> NS_TYPED_EXTENSIBLE_ENUM
```

Comment:
Why are changes to `common-AppKit.ignore` needed if we're excluding macOS and Mac Catalyst? This will just complicate the merge back to main later.

Reply:
Auto sanitize did that. I'll take a look

Reply:
Not updating the ignore will result in the following failures in xtro: https://gist.github.com/mandel-macaque/0e9384d87715fdc0f40604c5fd42cf35

Reply:
The correct fix is to move the ignores, I have created https://github.com/xamarin/xamarin-macios/issues/12769 to track a more appropriate fix.